.github/workflows/cron.yml

```
@@ -0,0 +1,30 @@
+name: cron
+
+on:
+  schedule:
+  # Run every day at 00:00 UTC
+  - cron: 0 0 * * *
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        python-version: [2.7, 3.5, 3.6, 3.7]
+        env:
+        - TOXENV: pipmaster
+        os: [ubuntu-latest, windows-latest]
+
+    steps:
+      - uses: actions/checkout@master
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
```

Don't need the pipe for single line command.
```suggestion
        run: pip install tox
```

---

Resolved. Thanks!